lib/pilot/build_manager.rb

```diff
@@ -9,7 +9,8 @@ def upload(options)
 
       package_path = PackageBuilder.new.generate(apple_id: app.apple_id,
                                                  ipa_path: config[:ipa],
-                                             package_path: "/tmp")
+                                             package_path: "/tmp",
+                                                 platform: "appletvos")
```

Comment:
This should obviously not be hardcoded, but I'm not sure what the best way to detect platform for the given ipa or app is.